packages/react-integration/demo-app-ts/src/components/demos/NumberInputDemo/NumberInputDemo.tsx

```diff
@@ -49,7 +49,7 @@ export class NumberInputDemo extends Component<NumberInputDemoState> {
 
   onPlus2 = () => {
     this.setState({
-      value2: this.state.value2 + 1
+      value2: this.state.value2 + 1 || 1
```

Comment:
This would make -1 skip 0 and go to 1. I wonder if something like `(this.state.value2 || 0) + 1` would be better.
Might not matter too much as this is for cypress.

Reply:
Yeah I can update it as that is clearer. I considered just removing the test because passing null doesn't behave as the test expects anymore (because it used to internally snap null to 0 but that prevented the input from ever being cleared for typing).